examples/feature_store/run.py

```diff
@@ -0,0 +1,115 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from datetime import datetime, timedelta
```

Comment:
I think this example needs a bit of rethink. 

* The function names like `print_historical_features` are perhaps too toy-like for a serious example
* Better to showcase a more realistic usage of feature storage data (i.e. to train a model)
* We do not need to show-case the online feature stuff in this example but rather in a deployment example later down the line

Reply:
I've removed the online showcase from the example. (We can add it later once we've added that functionality.)

For the training, the dataset is apparently not real and was just created for this Feast example (it seems?). I was unable to find any acknowledgement of where the data came from on the Feast site. There is [this example of training](https://colab.research.google.com/github/feast-dev/feast-driver-ranking-tutorial/blob/master/notebooks/Driver_Ranking_Tutorial.ipynb) but it seems to use a different dataset (since the columns are different).

I'd need help from someone to add in something where an example gets trained.